pkg/gornir/inventory.go

```diff
@@ -41,3 +46,22 @@ func (h *Host) SetErr(err error) {
 func (h *Host) Err() error {
 	return h.err
 }
+
+// SetConnections stores a connection
+func (h *Host) SetConnection(name string, conn Connection) {
+	if h.connections == nil {
+		h.connections = make(map[string]Connection)
+	}
+	h.connections[name] = conn
+}
```

Comment:
We need to define the behavior if you set a connection that already exist (name collision). Right now we are overwriting it, which might be OK but we need to al least warn the customer, otherwise thing might start failing silently.

Reply:
Agreed, opened #41 for this